lib/travis/build/script/r.rb

```diff
@@ -112,12 +112,13 @@ def configure
                   'cdbs qpdf texinfo libssh2-1-dev devscripts '\
                   "#{optional_apt_pkgs}", retry: true
 
-                r_filename = "R-#{r_version}-$(lsb_release -cs).xz"
-                r_url = "https://travis-ci.rstudio.org/#{r_filename}"
+                r_filename = "r-#{r_version}_1_amd64.deb"
+                os_version = "$(lsb_release -ds | perl -a -e '$F[1] =~ tr/[.]//d; print $F[1]')"
```

Comment:
This looks like bash... does this work in ruby?

Reply:
Yeah the entire thing gets inserted into the shell commands, so the `$()` does get interpreted by bash, not ruby.